source/cli.civet

```diff
@@ -322,11 +327,13 @@ export function repl(options: Options)
         if topLevelAwait
           // If there was a top-level await, the result is a promise
           // that we need to await before returning it.
+          let threw = false
           try
             result = await result
           catch error
+            threw = true
             callback error as Error, undefined
-          callback null, result
+          callback null, result unless threw
```

Comment:
isn't that the same as doing `callback null,result` inside the `try`?

Reply:
If the callback throws then the catch would also trigger if it were inside the `try`.